### DISCUSSION TITLE
Rasp PICO and CC1011 modules

### DISCUSSION THREAD

**Discussion by mastraa** [micropython/micropython#13669] [RP2040 / Pico]
Dear all,

is there any library to use CC1011 module (I bought a AS07-M11015 board) using MicroPython?

Best regards,
Andrea

**Comment by ned-pcs** [comment]
Have you looked at this: https://github.com/ilyamordasov/micropython-rcswitch

**Reply by mastraa** [reply]
Sorry ned,

I'm very new with MicroPython and Rasp Pico, I don't understand how to use files linked.
However it seems to work only for esp32 (I forgot to mention Pico, sorry) and also only for decoding and not sending, right?

Andrea

**Comment by ned-pcs** [comment]
Yes, that's correct. There does seem to be code at https://github.com/grspy/rcswitch-cc1101 that could serve as a guide to doing this with Micropython. You would use SPI to talk to the CC1101 chip.
